### PR TITLE
BZ2099314 updating procedure for accessing registry from the cluster

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
  * PR applies to all versions including and after: 4.6+

Issue:
https://bugzilla.redhat.com/show_bug.cgi?id=2099314

Link to docs preview:
http://file.rdu.redhat.com/shdiaz/BZ2099314/registry/accessing-the-registry.html#registry-accessing-directly_accessing-the-registry

Additional information:

### CODE REVIEW

**Review by bergerhoffer — COMMENTED** [review]
LGTM! Nice job @ShaunaDiaz :slightly_smiling_face:

File: modules/registry-accessing-directly.adoc [hunk 1]
```diff
@@ -22,7 +22,7 @@ $ oc get nodes
 +
 [source,terminal]
 ----
-$ oc debug nodes/<node_address>
+$ oc debug nodes/<node_name>
```
Comment (bergerhoffer) [inline modules/registry-accessing-directly.adoc:25]: Actually I just noticed that the step text above says "Access the node by getting the node's address". Should this be updated to instead say something about accessing the node by passing in the node name?

Comment (xiuwang) [reply modules/registry-accessing-directly.adoc:25]: +1

**Pull request merged**